WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -806,7 +857,7 @@ private extension ProductFormViewController {
         defer {
             navigationController?.popViewController(animated: true)
         }
-        let hasChangedData = newBriefDescription != product.briefDescription
+        let hasChangedData = newBriefDescription != product.shortDescription
```

Comment:
nit: rename `newBriefDescription" to `newShortDescription`

Reply:
yup! updated in https://github.com/woocommerce/woocommerce-ios/pull/2565/commits/0c07513322a54454c42e4bf13688164c355c1454